deploy.sh

```diff
@@ -15,13 +15,13 @@ cargo doc
 
 echo -e "${CYAN}Running mdbook build${NC}"
 # Run mdbook to generate the book
-target/debug/mdbook build book-example/
+target/$TARGET/debug/mdbook build book-example/
```

Comment:
note: we need target/$TARGET instead of just target in all these instances because mdbook was compiled with `cargo build --target $TARGET` instead of just `cargo build`